modules/installation-gcp-user-infra-config-host-project-vpc.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-gcp-user-infra-config-host-project-vpc_{context}"]
+= Configuring the GCP project that hosts your shared VPC network
+
+If you use a shared Virtual Private Cloud (VPC) to host your {product-title} cluster in Google Cloud Platform (GCP), you must configure the project that hosts it.
+
+[NOTE]
+====
+If you already have a shared VPC, review this section to ensure that the project that hosts your shared VPC meets all of the requirements to install a {product-title} cluster.
```

Comment:
If I understand correctly, you want to talk about the existing host project. Then I suggest to change it to:
If you already have a project that hosts shared VPC network, review this section to ensure that the project meets ...